Tests/SwiftDocCTests/Infrastructure/ExternalReferenceResolverTests.swift

```diff
@@ -1434,4 +1434,56 @@ class ExternalReferenceResolverTests: XCTestCase {
         }
         XCTAssertEqual(externalLinkCount, 2, "Did not resolve the 2 expected external links.")
     }
+
+    func testPossibleValuesWithExternalLink() throws {
+
+        // Create some example documentation using the symbol graph file located under
+        // Tests/SwiftDocCTests/Test Bundles/DictionaryData.docc, and the following
+        // documentation extension markup.
+        let documentationExtension = TextFile(
+            name: "Genre.md",
+            utf8Content: """
+                    # ``DictionaryData/Genre``
+                    
+                    Genre object.
+                    
+                    The artist's genre.
+                    
+                    - PossibleValues:
+                      - Classic Rock: Something about classic rock with a link: <doc://com.external.testbundle/something/related/to/this/allowed/value>.
+                      - Folk: Something about folk music with a link: <doc://com.external.testbundle/something/related/to/this/allowed/value>.
+                    """)
+        let symbol = try exampleDocumentation(
+            copying: "DictionaryData",
+            documentationExtension: documentationExtension,
+            path: "/documentation/DictionaryData/Genre"
+        )
+
+        // Get the variant of the example symbol that has no interface language, meaning it was
+        // generated by the markup above.
+        let variant = symbol.possibleValuesSectionVariants.allValues.first(
+            where: { $0.trait == .init(interfaceLanguage: nil) }
+        )
+
+        let section = try XCTUnwrap(variant?.variant)
```

Comment:
`nil` is the only interface language for this symbol, so this `.first(where:)` statement is redundant here.

```suggestion
        let section = try XCTUnwrap(symbol.possibleValuesSectionVariants.firstValue)
```